src/pages/CustomerInvoicePage.js

```diff
@@ -205,6 +171,19 @@ export const CustomerInvoicePage = ({ transaction, runWithLoadingIndicator, rout
     />
   );
 
+  const getModalOnSelect = () => {
+    switch (modalKey) {
+      case ITEM_SELECT:
+        return value => dispatch(editComment(value, 'Transaction'));
```

Comment:
There miiiight be a chance you didn't test adding items aye 😛

Reply:
daf85c90cb54d5abd7a278bf437d0b2324da4418